src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -83,6 +83,28 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return BackgroundImagePath() != L"";
     }
 
+    Editor::ColorSchemeViewModel AppearanceViewModel::CurrentColorScheme()
```

Comment:
Make sure the `reset` button works properly! I encountered an issue when I tried this out.

Reply:
Thanks for catching that! Fixed.